R/expr.R

```diff
@@ -30,7 +30,7 @@
 #' @export
 #' @examples
 #' # expr_eval() works just like base::eval():
-#' env <- env_new(dict = list(foo = "bar"))
```

Comment:
I wonder if it's worth having `as_env()`? Then this could be `env <- as_env(list(foo = "bar"))`